docs/apis-tools/grpc.md

```diff
@@ -776,6 +776,78 @@ Returned if:
 - The instructions would terminate all element instances of a process instance that was created by a call activity in
   the parent process.
 
+## `MigrateProcessInstance` RPC
+
+Migrates a process instance to a new process definition. The command can contain multiple mapping instructions
+to define mapping between the active process instance's elements and target process definition elements.
+
+Use the command to upgrade a process instance to a new version of a process or to a different process definition.
+E.g. keep your running instances up-to-date with the latest process improvements.
+
+:::note
```

Comment:
@korthout I am not sure if we should mention this part here. I think, it would be better if we give reference to the document (related issue https://github.com/camunda/camunda-docs/issues/2900) where we mention the concept and its limitations. WDYT?